src/client/containers/ProductPage/ProductPage.Style.css

```diff
@@ -1,3 +1,13 @@
-h1 {
-  color: aqua;
+.p-detail {
+  margin: 50px auto;
+  width: 80%;
+}
```

Comment:
If you change this part with this codes then product details component will be on center of the page.
`.p-detail {
  padding: 50px 0;
  display: flex;
  justify-content: center;
}`

Reply:
ok

Reply:
You probably want to address this in app.js. Have a wrapper around the entire site with a max-width (perhaps 1400 or 1600px) and set that wrapper to be centered. Generally components should be full-width. This way the wrapper will control the width of everything on the site.